### PR TITLE
Mass report update - FLOPS

### PR DESCRIPTION
**Pull request by jkirk5**
### Summary

Updates mass reporting for FLOPS-based mass to conform to SAWE RP-8 standard.
Some mass variables have been renamed to comply with the standard, and the FLOPS-based mass summation component was reworked to group masses appropriately.

Generative AI was used to write a new test that checks matching between hierarchy variable names and string representation (test_var_structure.py). The Cursor IDE was used, which does not expose which specific models it uses. Manual review and minor tweaks were done by hand.

### Related Issues

- Halfway to #71

### Backwards incompatibilities
Renamed variables:
`Aircraft.CrewPayload.NUM_TOURIST_CLASS` -> `Aircraft.CrewPayload.NUM_ECONOMY_CLASS`
`Aircraft.CrewPayload.NON_FLIGHT_CREW_MASS` -> `Aircraft.CrewPayload.CABIN_CREW_MASS`
`Aircraft.CrewPayload.NON_FLIGHT_CREW_MASS_SCALER` -> `Aircraft.CrewPayload.CABIN_CREW_MASS_SCALER`
`Aircraft.CrewPayload.PASSENGER_MASS` -> `Aircraft.CrewPayload.PASSENGER_MASS_TOTAL`
`Aircraft.CrewPayload.PASSENGER_MASS_WITH_BAGS` -> `Aircraft.CrewPayload.MASS_PER_PASSENGER_WITH_BAGS`
`Aircraft.Controls.TOTAL_MASS` -> `Aircraft.Controls.MASS`
`Aircraft.Design.SYSTEMS_EQUIP_MASS` -> `Aircraft.Design.SYSTEMS_AND_EQUIPMENT_MASS`
`Aircraft.Design.SYSTEMS_EQUIP_MASS_BASE` -> `Aircraft.Design.SYSTEMS_AND_EQUIPMENT_MASS_BASE`
`Aircraft.Wing.ASPECT_RATIO_REF` -> `Aircraft.Wing.ASPECT_RATIO_REFERENCE`
`Aircraft.Wing.THICKNESS_TO_CHORD_REF` -> `Aircraft.Wing.THICKNESS_TO_CHORD_REFERENCE`

### New Dependencies

None

### CODE REVIEW

File: aviary/models/aircraft/large_single_aisle_2/large_single_aisle_2_altwt_FLOPS_data.py [hunk 1]
```diff
@@ -334,6 +336,7 @@
 
 outputs.set_val(Aircraft.LandingGear.MAIN_GEAR_MASS, 0.85 * 5778.0, 'lbm')
 outputs.set_val(Aircraft.LandingGear.NOSE_GEAR_MASS, 0.15 * 5778.0, 'lbm')
+outputs.set_val(Aircraft.LandingGear.TOTAL_MASS, 4911.3 + 866.7, 'lbm')
```
Comment (Kenneth-T-Moore) [inline aviary/models/aircraft/large_single_aisle_2/large_single_aisle_2_altwt_FLOPS_data.py]: Let's just make this the 5778.0 from the above 2 lines....

File: aviary/subsystems/mass/flops_based/landing_gear.py [hunk 2]
```diff
@@ -62,6 +63,16 @@ def setup_partials(self):
                 Aircraft.Design.TOUCHDOWN_MASS,
             ],
         )
+        self.declare_partials(
```
Comment (Kenneth-T-Moore) [inline aviary/subsystems/mass/flops_based/landing_gear.py]: There is a problem here if we override one of the gear masses, but let Aviary compute the other (I know, an odd corner case, but it is possible). In that case, the total will be wrong. To overcome this, best to implement the total as a separate component.

File: aviary/subsystems/mass/flops_based/test/test_mass_summation.py [hunk 3]
```diff
@@ -152,8 +157,9 @@ def test_case(self, case_name):
                 Aircraft.HorizontalTail.MASS,
                 Aircraft.Hydraulics.MASS,
                 Aircraft.Instruments.MASS,
-                Aircraft.LandingGear.MAIN_GEAR_MASS,
-                Aircraft.LandingGear.NOSE_GEAR_MASS,
+                # Aircraft.LandingGear.MAIN_GEAR_MASS,
+                # Aircraft.LandingGear.NOSE_GEAR_MASS,
```
Comment (Kenneth-T-Moore) [inline aviary/subsystems/mass/flops_based/test/test_mass_summation.py]: We should clean up all these commented out keys. It is okay to delete them, since we have version control.
